store/setup.ts

```diff
@@ -64,75 +67,100 @@ const initialState: SetupState = {
   approvedRequestsCount: null,
 };
 
+const getRankedUsers = (leaderboard: LeaderboardUser[]): LeaderboardUser[] => {
```

Comment:
I think it should automatically be LeaderboardUser, since the response is returning rankedUsers.
```suggestion
const getRankedUsers = (leaderboard: LeaderboardUser[]) => {
```

Reply:
it does, but it's a good practice to define return type to ensure nothing else is returned

Reply:
Sure! Just my 2 cents: you can actually hover over the function, and TypeScript is smart enough to infer the type automatically—this is known as implicit typing.